### ISSUE TITLE
Pip install not working on version 1.6.1 and Python 3

### ISSUE THREAD

**Issue by bulv1ne** [jazzband/django-pipeline#522]
When installing ``django-pipeline`` on Python 3.5 on OSX, it tries to install ``subprocess32`` as well, which is only available for Python 2.x

Output of pip:
```
$ pip install django-pipeline
Collecting django-pipeline
  Using cached django_pipeline-1.6.1-py2.py3-none-any.whl
Collecting futures>=2.1.3 (from django-pipeline)
Collecting subprocess32>=3.2.7 (from django-pipeline)
  Using cached subprocess32-3.2.7.tar.gz
    Complete output from command python setup.py egg_info:
    This backport is for Python 2.x only.

    ----------------------------------------
```

**Comment by bulv1ne** [comment]
After looking more into it, it only fails for the ``.whl`` file, but the ``.tar.gz`` works fine

**Comment by cyberdelia** [comment]
Urg. Seems like this need a wheel specific for Python 3.

**Comment by cyberdelia** [comment]
Should be fixed in 1.6.2, sorry about that.
